accounts-cluster-bench/src/main.rs

```diff
@@ -586,6 +669,7 @@ fn run_accounts_bench(
     loop {
         if latest_blockhash.elapsed().as_millis() > 10_000 {
             blockhash = poll_get_latest_blockhash(&client).expect("blockhash");
+            slot_height_atom.store(poll_slot_height(&client), Ordering::Relaxed);
```

Comment:
This means the slot height will be updated every 10 seconds, along with the blockhash. This is super dumb, but probably good enough.